pacta_data_qa.Rmd

```diff
@@ -0,0 +1,678 @@
+---
+title: "PACTA Analysis - Input Data QA"
+author: "Jacob Kastl"
+date: "`r format(Sys.time(), '%B %Y')`"
+mainfont: GT America
+params: 
+  project_code: general
+  project_location_ext: NULL
+  twodii_internal: TRUE
+output:
+  html_document:
+    fig_caption: yes
+    theme: flatly
+    toc: yes
+    toc_depth: 3
+    toc_float: yes
+---
+
+```{r setup, include=FALSE}
+# set echo = TRUE, for more insight to the code printed in the output
+knitr::opts_chunk$set(echo = FALSE)
+library(tidyverse)
+library(DT)
+# source("1_portfolio_check_initialisation.R")
+
+# devtools::load_all()
+# use_r_packages()
+# ## Project Initialisation
+# working_location <- paste0(getwd(), "/")
```

Comment:
can we remove commented code like this, rather than carry it forward with us indefinitely

Reply:
Sure. this is one case, where we need to improve the first initialise script of PACTA_analysis offline so that we can just source it or call a function rather than setting these all the time.